### PR TITLE
dotCMS/core#15910 SPAEM: Create news detail page

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: src/Components/News/NewsListItem.js [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
     return (
         <Media className="news-list-row" tag="li">
             {displayedFields.includes('image') ? (
-                <Media left href={`/news-events/news/${news.identifier}`}>
-                    <Media object src={news.image} alt="image" />
-                </Media>
+                <Link
+                    to={{
+                        pathname: `/news-events/news/${news.urlTitle}`,
+                        state: { news: news }
```
[the remark this answers is not in the document]
Comment (alfredo-dotcms) [reply src/Components/News/NewsListItem.js]: https://forum.wordreference.com/threads/singular-form-for-news.1067564/

File: src/Components/News/NewsDetail.js [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
 
-const NewsDetail = ({ news }) => {
-    const { title, sysPublishDate, image, story } = news;
+const NewsDetail = ({ title, sysPublishDate, image, story }) => {
     return (
         <>
             <h3>{title}</h3>
             <h5>{sysPublishDate}</h5>
-            <img alt="" src={image} />
+            <img className='news-detail-img' alt={title} src={image} />
```
Comment (fmontes) [inline src/Components/News/NewsDetail.js]: HTML attr double quoted.

File: src/Components/News/NewsListItem.js [hunk 3]
```diff
@@ -1,24 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Media } from 'reactstrap';
+import { Link } from 'react-router-dom';
+
+const Links = props => {
```
Comment (fmontes) [inline src/Components/News/NewsListItem.js]: This are not links, there more like `ItemLink`

File: src/Components/News/NewsListItem.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (fmontes) [reply src/Components/News/NewsListItem.js]: 😱 didn't know that.

**Pull request merged**